code/game/objects/items.dm

```diff
@@ -116,6 +124,16 @@ GLOBAL_VAR_INIT(rpg_loot_items, FALSE)
 	if(GLOB.rpg_loot_items)
 		rpg_loot = new(src)
 
+	LAZYINITLIST(default_strings)
+	//populate default_strings
+	default_strings += "very low"
+	default_strings += "low"
+	default_strings += "medium"
+	default_strings += "high"
+	default_strings += "robust"
+	default_strings += "very robust"
+	default_strings += "exceptionally robust"
+

```

Comment:
Here's a better way to do this, use the default strings in the switch directly. Then, add a normally null list to /item called force_description_strings and use that instead of the defaults if it exists